src/predict_request_validation_utils.cpp

```diff
@@ -74,7 +73,71 @@ class RequestValidator {
     Status validate();
 };
 
-Status RequestValidator::validateNumberOfInputs(const tensorflow::serving::PredictRequest& request) const {
+class RequestValidatorKFS {
```

Comment:
We could have RequestValidatorKFS and RequestValidatorTF that inherit from interface RequestValidator